### PR TITLE
UNIFY-412 update example recipe readme and examples

### PR DESCRIPTION
**Pull request by jaffrepaul**
# DO NOT MERGE

This PR **will not be merged** but its content will be folded into an upcoming release. This contains many (but not all) of the updates needed to document the `cypress.json` -> `cypress.config.js` changes.

https://cypress-io.atlassian.net/browse/UNIFY-412

**Note:** batch lint files after review & before merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,8 +131,8 @@ Recipe | Description
 [Seeding your Database in Node](./examples/server-communication__seeding-database-in-node) | Seed your database with test data
 [XHR assertions](./examples/server-communication__xhr-assertions) | Spy and assert on application's network calls
 [Visiting 2nd domain](./examples/server-communication__visit-2nd-domain) | Visiting two different domains from two different tests and passing value from one test to another
-[Pass value between specs](./examples/server-communication__pass-value-between-specs) | Pass a value from spec to spec via the plugin file using [cy.task](https://on.cypress.io/task)
-[Stream test results](./examples/server-communication__stream-tests) | Streams each test result from the browser to the plugins to an external process via IPC
+[Pass value between specs](./examples/server-communication__pass-value-between-specs) | Pass a value from spec to spec via the setupNodeEvents function in the Cypress configuration using [cy.task](https://on.cypress.io/task)
```
Comment (lmiller1990) [inline README.md:134]: Might be good to make `setupNodeEvents` have backticks to emphasize it is a specific function that is part of the configuration.

**Review by lmiller1990 — COMMENTED** [review]
Are all these changes intentional? It looks like some formatting has been applied to everything - not sure if this was intentional or not, but makes it kind of hard to review.

Probably okay for now but would be good to enforce some automated linting across our examples repo.

**Comment by cowboy** [comment]
> Are all these changes intentional? It looks like some formatting has been applied to everything - not sure if this was intentional or not, but makes it kind of hard to review.

Every `cypress.json` + plugins file had to be merged and reformatted before we started going through them, are you talking about those files, or something else?

**Comment by cowboy** [comment]
> Every `cypress.json` + plugins file had to be merged and reformatted before we started going through them, are you talking about those files, or something else?

I see what you're saying now. @jaffrepaul what's the LOE to split this into 2 PRs, one that has just the formatting changes (quotes, whitespace), and another from that PR that has all the other changes?

File: README.md (hunk 1 above)
Comment (jaffrepaul) [reply README.md:134]: good catch, I'll update these.

**Comment by jaffrepaul** [comment]
@cowboy @lmiller1990 No problem on the linting. I ran the command at the end to get everything consistent. I'll split that off and we can handle separately.

**Comment by jaffrepaul** [comment]
OK @lmiller1990 @cowboy I reverted the linting & added `setUpNodeEvents` for clarity. I kept them as two commits for clarity. 

Note: 
- most of the linting was batched via the eslint command but some was picked up via the Husky pre commit hook while I was editing some files by hand. I reverted these too, which in isolation felt a little silly. But there's far less noise in this PR now and I can just scoop all these up with the command separately. There are a 1-2 files that are tracking the style revert as a result of being included with other stuff. Just know that will all be rectified later. 

- in the interest of time I wrapped `setUpNodeEvents` in .md files and comments as well. I can edit that if it's weird

**Pull request merged**